.github/workflows/ci.yaml

```diff
@@ -37,6 +37,27 @@ jobs:
         run: hack/verify-all.sh -v
         env:
           GO111MODULE: auto
+      
+      - name: golangci-lint
```

Comment:
```suggestion
      - name: golangci-lint - balancer
```

Reply:
The Github Actions output isn't clear about which job failed:
<img width="228" alt="image" src="https://github.com/user-attachments/assets/cbce00b2-d905-4675-9b14-570b33ccdd79">